testassets/LinkerTestsClient/LinkerTestsClient.csproj

```diff
@@ -1,29 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <OutputType>Exe</OutputType>
-    
     <PublishTrimmed>true</PublishTrimmed>
-    <TrimMode>Link</TrimMode>
+
+    <!-- IL2104: Other assemblies have warnings -->
```

Comment:
Which assemblies are these? Might help to document in case we can eventually remove this suppression.